Get-NetView.psm1

```diff
@@ -1901,6 +1901,42 @@ function NetworkSummary {
     ExecCommandsAsync -OutDir $dir -File $file -Commands $cmds
 } # NetworkSummary()
 
+function PktmonDetail {
+    [CmdletBinding()]
+    Param(
+        [parameter(Mandatory=$true)] [String] $OutDir
+    )
+
+    $dir = (Join-Path -Path $OutDir -ChildPath "Pktmon")
+    New-Item -ItemType directory -Path $dir | Out-Null
+
+    $file = "pktmon.status.txt"
+    [String []] $cmds = "pktmon status",
+                        "pktmon stop" # End any existing session
+    ExecCommands -OutDir $dir -File $file -Commands $cmds
+
+    $file = "pktmon.filter.txt"
+    [String []] $cmds = "pktmon filter list"
+    ExecCommands -OutDir $dir -File $file -Commands $cmds
+
+    $file = "pktmon.list.txt"
+    [String []] $cmds = "pktmon list",
+                        "pktmon list --all",
+                        "pktmon list --all --include-hidden"
+    ExecCommands -OutDir $dir -File $file -Commands $cmds
+
+    # Reset state
+    $null = pktmon unload
+
+    $file = "pktmon.counters.txt"
+    [String []] $cmds = "pktmon start --capture --counters-only --comp all",
+                        "Start-Sleep 1",
```

Comment:
This looks like it's a simple test to sample if the network was active.  A comment would be helpful.